biojava-structure/src/main/java/org/biojava/nbio/structure/SVDSuperimposer.java

```diff
@@ -131,11 +131,11 @@
  */
 public class SVDSuperimposer {
```

Comment:
SVDSuperimposer actually uses double arrays and jama matrices internally. Atoms are only used by the constructor as inputs, and then converted to double arrays. The transformation matrix is returned as Matrix4d.
I already thought about implementing a constructor for Point3d, which will be converted to double arrays like the Atom constructor. That seems like something we can easily do.
I did not move it to geometry package because it is used in a lot of places and would require a major change. I think it is better to move it than duplicating it,though.